shared/chat/inbox/row/index.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React from 'react'
+import BigTeamHeader from './big-team-header/container'
+import BigTeamChannel from './big-team-channel/container'
+import FilterBigTeamChannel from './filter-big-team-channel/container'
+import FilterSmallTeamChannel from './filter-small-team/container'
+import SmallTeam from './small-team/container'
+import * as Constants from '../../../constants/chat'
+
+type MakeRowOptions = {
+  channelname: string,
+  conversationIDKey: ?Constants.ConversationIDKey,
+  filtered: boolean,
+  isActiveRoute: boolean,
+  teamname: string,
+  type: 'small' | 'bigHeader' | 'bigTeamsLabel' | 'big' | 'divider',
```

Comment:
we don't need `bigTeamsLabel` or `divider` here, right?

Reply:
i moved bigTeamsLabel into here and removed divider, thanks!